modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -30,6 +30,23 @@
 
 The following premium plugin updates were released alongside {productname} 6.5.
 
+=== Checklist 2.0.5
+
+The {productname} 6.5 release includes an accompanying release of the **Checklist** premium plugin.
+
+**Checklist** 2.0.5 includes one fix:
+
+=== Checklist toggling was possible on `contenteditable = "false"` elements
+//#TINY-9469
+
+In earlier versions of {productname}, the Checklist plugin allowed toggling on elements with the `contenteditable = "false"` attribute.
+
+As a consequence, lists nested within a `contenteditable = "false"` section of the editor could be converted into a checklist.
+
+To address this issue, the Checklist button has now been disabled for `contenteditable = "false"` sections in the {productname} 6.5.
```

Comment:
@hamza0867, can someone still able to programmatically toggle contenteditable=false elements via commands?